### PR TITLE
[DDW-1133]: Update package dependencies

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR updated package dependencies to prevent vulnerability issues
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-1133)

## Todos
- [x] Update package dependencies

## Testing Checklist
- [x] Ensure app is working properly

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12540,15 +12395,16 @@ term-size@^1.2.0:
   dependencies:
     execa "^0.7.0"
 
-terser-webpack-plugin@^1.1.0, terser-webpack-plugin@^1.2.4, terser-webpack-plugin@^1.4.1:
-  version "1.4.1"
-  resolved "https://registry.yarnpkg.com/terser-webpack-plugin/-/terser-webpack-plugin-1.4.1.tgz#61b18e40eaee5be97e771cdbb10ed1280888c2b4"
+terser-webpack-plugin@1.4.2, terser-webpack-plugin@^1.2.4, terser-webpack-plugin@^1.4.1:
+  version "1.4.2"
+  resolved "https://registry.yarnpkg.com/terser-webpack-plugin/-/terser-webpack-plugin-1.4.2.tgz#e23c0d554587d1f473bd0cf68627720e733890a4"
+  integrity sha512-fdEb91kR2l+BVgES77N/NTXWZlpX6vX+pYPjnX5grcDYBF2CMnzJiXX4NNlna4l04lvCW39lZ+O/jSvUhHH/ew==
```
Comment (nikolaglumac) [inline yarn.lock]: @yakovkaravelov please run `yarn lockfile:fix` to remove all of the `integrity` lines. This is a must as having these lines in the yarn.lock file breaks the CI builds.

Comment (yakovkaravelov) [reply yarn.lock]: @nikolaglumac Fixed. Thanks.

**Comment by nikolaglumac** [comment]
@DmitriiGaico no QA required for this one.

**Comment by yakovkaravelov** [comment]
There was 1 package which was dependent on `serialize-javascript`.
That packages was `terser-webpack-plugin`.
I looked into release versions of this pacakge and found out that v.1.4.2 fixed dependency of `serialize-javascript` to use v.2.1.1 for vulnerabilities issues fix.
And there were 2 packages which were dependent on `terser-webpack-plugin`.
There were `@storybook/core`, and `webpack`.
So I looked into releases of these 2 pacakges, and found out
1. Later `webpack` package was dependent on v1.4.1 of `terser-webpack-plugin`
   That's why I decided to update `webpack` to later version which is 4.39.1
2. `storybook` package was dependent on v1.2.4 of `terser-webpack-plugin` and only beta version of storybook was dependent on later version of `terser-webpack-plugin`, so I decided to leave this as it is.

So finally I added `"**/**/terser-webpack-plugin": "1.4.2"` entry to resolution section of pacakge.json

**Pull request merged**